Elasticsearch - Cannot call logger

<!--- Provide a general summary of the issue in the **Title** above -->
<!--- Before you open an issue, please check if a similar issue already exists or has been closed before. --->
<!--- If you have discovered a security risk please report it by emailing security@suitecrm.com. This will be delivered to the product team who handle security issues. Please don't disclose security bugs publicly until they have been handled by the security team. --->

#### Issue
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
Missing a `use` statement in `SearchResults.php` - causes the following error:

```
FATAL] [ERROR] [SearchThrowableHandler] Error: Class 'SuiteCRM\Search\LoggerManager' not found in /var/www/html/QA/lib/Search/SearchResults.php:195
Stack trace:
#0 /var/www/html/QA/lib/Search/SearchResults.php(169): SuiteCRM\Search\SearchResults->getRelatedId(Object(aCase), 'aos_product_cat...', 'cases_aos_produ...')
#1 /var/www/html/QA/lib/Search/SearchResults.php(153): SuiteCRM\Search\SearchResults->updateObjLinks(Object(aCase), Array)
#2 /var/www/html/QA/lib/Search/SearchResults.php(135): SuiteCRM\Search\SearchResults->updateFieldDefLinks(Object(aCase), Array)
#3 /var/www/html/QA/lib/Search/UI/SearchResultsController.php(122): SuiteCRM\Search\SearchResults->getHitsAsBeans()
```
#### Expected Behavior
<!--- Tell us what should happen -->
Results are displayed
#### Actual Behavior
<!--- Tell us what happens instead -->
<!--- Also please check relevant logs (suitecrm.log, php error.log etc.) -->
User is alerted of an error and NO results are shown
#### Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->
See attached PR
#### Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug include code to reproduce, if relevant -->
You have to somehow have an invalid record. In my case, it happened automatically by Indexing the Cases module. There's something amis between cases and categories. But I'm not sure if this is a general error or specific to my customized instance. I believe there are other tickets related to the Cases module.

1. Somehow prepare an invalid record (maybe as easy as create a Case)
2. Configure and Index Elasticsearch
3. Perform a search that would include your record
4.

#### Context
<!--- How has this bug affected you? What were you trying to accomplish? -->
<!--- If you feel this should be a low/medium/high priority then please state so -->
Trying to search

#### Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* SuiteCRM Version used: 7.11.8
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):Chrome
* Environment name and version (e.g. MySQL, PHP 7):LAMP
* Operating System and version (e.g Ubuntu 16.04):LAMP

--- a/lib/Search/SearchResults.php
+++ b/lib/Search/SearchResults.php
@@ -39,9 +39,10 @@
 
 namespace SuiteCRM\Search;
 
-use BeanFactory;
-use RuntimeException;
 use SugarBean;
+use BeanFactory;
+use LoggerManager;
+use RuntimeException;
 use SuiteCRM\Exception\Exception;
 
 if (!defined('sugarEntry') || !sugarEntry) {

